keras_trainer/callbacks.py

```diff
@@ -0,0 +1,49 @@
+import os
+import numpy as np
+
+from keras.callbacks import Callback
+from sklearn.metrics import recall_score
+
+
+class SensitivityCallback(Callback):
+    def __init__(self, validation_data_generator, output_model_dir, batch_size=10):
+        super(SensitivityCallback, self).__init__()
+        self.validation_data_generator = validation_data_generator
+        self.batch_size = batch_size
+        self.output_model_dir = os.path.join(output_model_dir, 'model_max_sensitivity.hdf5')
+        self.max_sensitivity = -1
+
+    def on_epoch_end(self, epoch, logs={}):
+        n_samples = self.validation_data_generator.samples
+        n_batches = n_samples // self.batch_size
+        last_batch_remainder = n_samples % self.batch_size
+
+        y_true = []
+        y_pred = []
+
+        # Forward pass and store labels and predictions
+        for i, batch in enumerate(self.validation_data_generator):
+            if i == n_batches and last_batch_remainder > 0:
+                batch = batch[0][0:last_batch_remainder], batch[1][0:last_batch_remainder]
+
+            elif i > n_batches:
+                break
+
+            y_true += np.argmax(batch[1], axis=1).tolist()
```

Comment:
Do we need the `argmax()` call here? Think `recall_score` and take in a matrix (not sure tho?). The main reason is that we might want to extend this to compute other metrics that need the probabilities (e.g., top-k sensitivity, account for probabilistic labels, etc)

Reply:
*** ValueError: Classification metrics can't handle a mix of multilabel-indicator and continuous-multioutput targets ***

Looks like it won't work